data/human/variants.txt

```diff
@@ -11,6 +11,27 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+ship "Aerie" "Aerie (Derelict)"
+	"uncapturable"
```

Comment:
Seeing as these ships are uncapturable, why change their stats? I understand the inclusion of automaton and the bunk and required crew changes to force their crews to zero, but why change the cost, hull, outfit space, hardpoints, etc?

Reply:
tbf with the cost based boarding method lower price means lower priority to board, compared to other ships

as for the rest I agree there's no point, they're already shown as damaged enough since they're derelicts

Reply:
They already have less outfits, so that also reduces their priority.